container/kvm/container.go

```diff
@@ -28,9 +30,20 @@ func (c *kvmContainer) Name() string {
 }
 
 func (c *kvmContainer) Start(params StartParams) error {
-
-	logger.Debugf("Synchronise images for %s %s %v", params.Series, params.Arch, params.ImageDownloadURL)
-	if err := SyncImages(params.Series, params.Arch, params.ImageDownloadURL); err != nil {
+	var srcFunc func() simplestreams.DataSource
+	if params.ImageDownloadURL != "" {
+		srcFunc = func() simplestreams.DataSource {
+			return imagedownloads.NewDataSource(params.ImageDownloadURL)
+		}
+	}
+	sp := SyncParams{
+		arch:    params.Arch,
```

Comment:
unexported params it implements an interface, so it doesn't' need exporting. Sync may not actually require it either, but this was previously exported.